KomaMRIIO/src/KomaMRIIO.jl

```diff
@@ -0,0 +1,21 @@
+module KomaMRIIO
+
+using KomaMRICore
+using Scanf, FileIO, HDF5, MAT, JLD2    # IO related
+using MRIBase                           # At least for EncodingCounters()
+using MRIFiles                          # At least for Limits()
```

Comment:
If KomaCore has MRIBase this is also needed?

Reply:
Addessed in [this commit](https://github.com/cncastillo/KomaMRI.jl/commit/210c5172fec1bb2351b0419f1bac3a619814a1bc).